src/import/importctl.c

```diff
@@ -908,6 +908,11 @@ static int list_images(int argc, char *argv[], void *userdata) {
         (void) table_set_sort(t, (size_t) 0, (size_t) 1);
         table_set_ersatz_string(t, TABLE_ERSATZ_DASH);
 
+        /* Starting in v257, these fields would be automatically formatted with underscores. However, this
+         * command was introduced in v256, so changing the field name would be a breaking change. */
+        (void) table_set_json_field_name(t, 8, "usage-exclusive");
+        (void) table_set_json_field_name(t, 10, "limit-exclusive");
```

Comment:
I don't feel the necessity to do this - we don't promise the same level of compatibility as public APIs for CLI tools' JSON output (e.g. https://github.com/systemd/systemd/pull/30958), and if some json parsers don't like dashes we'd better translate them here too.

Reply:
well, our json output only really exists to be consumable/parsable by other tools. To make this workable we really should try to make it stable (unless it's so broken that it was unusable before)

Reply:
so should I put this back?

Reply:
I think that would be best, yes.

Reply:
FWIW, I think it's fine if we change the naming. Not in a stable release, but in a new upstream release. I think the number of people who use this and are not upstream developers is miniscule. If  you decide to keep the override, I'm fine with that too ;)